compiler/modules/front/resolveTypes.bal

```diff
@@ -278,6 +278,16 @@ function resolveTypeDesc(ModuleSymbols mod, s:ModuleLevelDefn modDefn, int depth
         }
         return t:xmlSequence(t);
     }
+    if td is s:TableTypeDesc {
+        t:SemType t = check resolveTypeDesc(mod, modDefn, depth, td.row);
+        d:Location loc =  d:location(modDefn.part.file, td.startPos, td.endPos);
```

Comment:
Put the declaration of `loc` inside the `if`, and do the same for the XML case above.